Lib/fontTools/cffLib/__init__.py

```diff
@@ -1660,7 +1673,7 @@ def xmlRead(self, name, attrs, content, parent):
 
 class VarStoreConverter(SimpleConverter):
```

Comment:
you forgot this one?

Reply:
According to your next comment I will update SimpleConverter itself.